backend/functions/feedback/feedback.py

```diff
@@ -0,0 +1,50 @@
+"""
+url for local testing:
+http://127.0.0.1:5001/schemessg-v3-dev/asia-southeast1/feedback
+"""
+
+from firebase_functions import https_fn
+from firebase_admin import firestore
+from uuid import uuid4
+from datetime import timezone 
+import datetime 
+
+# Firestore client
+db = firestore.client()
+
+# Firestore collection name
+COLLECTION_NAME = "schemeEntries"
+
+@https_fn.on_request(region="asia-southeast1")
+def feedback(req: https_fn.Request) -> https_fn.Response:
+    if req.method != "POST":
+        return https_fn.Response("Only POST requests are allowed.", status=405)
+
+    try:
+        # Parse the request data
+        request_json = req.get_json()
+        feedback_text = request_json.get("feedbackText")
+        timestamp = datetime.datetime.now(timezone.utc) 
+
+        if not feedback_text or not timestamp:
+            return {"success": False, "message": "Missing required fields."}, 400
+
+        # Generate a UUID
+        request_uuid = str(uuid4())
+
+        # Prepare the data for Firestore
+        feedback_data = {
+            "feedbackText": feedback_text,
+            "timestamp": timestamp,
+            "requestUUID": request_uuid,
```

Comment:
please check the metadata of existing document in feedback collection to align on the fields
probably can add name and email instead of userId
- User Feedback (userFeedback) - Created
    
    feedbackText
    
    timestamp
    
    userId

Reply:
I've updated the sample schema in firebase console for the userFeedback collection to align with what was detailed in the notion documentation https://www.notion.so/better/Schemes-V3-Architecture-4f709f7dfe5842ebb7e947f885b08516

<img width="1374" alt="image" src="https://github.com/user-attachments/assets/43148820-06a6-49a9-b771-b9eddcce1c9b">